dianna/methods/shap.py

```diff
@@ -1,21 +1,223 @@
 import numpy as np
+import shap
+import onnx
+from skimage.segmentation import slic
```

Comment:
```suggestion
import skimage.segmentation
```

Reply:
Thanks for your comment @egpbos . It has been changed now.